stdlib/REPL/src/REPL.jl

```diff
@@ -194,7 +194,9 @@ function modules_to_be_loaded(ast::Expr, mods::Vector{Symbol} = Symbol[])
         end
     end
     for arg in ast.args
-        arg isa Expr && modules_to_be_loaded(arg, mods)
+        if arg isa Expr && arg.head in [:block, :if, :using, :import]
```

Comment:
```suggestion
        if arg isa Expr && arg.head in (:block, :if, :using, :import)
```

The `[]` form allocates the same tuple as above, then splats it to a new array, on every iteration of the loop.

Reply:
Or even more concisely:
```suggestion
        if isexpr(arg, (:block, :if, :using, :import))
```

Reply:
Makes sense, thanks https://github.com/JuliaLang/julia/pull/43708